### PR TITLE
Document all ways to start a new project

### LINKED ISSUE #1824: Document all ways to start a new project
* https://code.quarkus.io/
* Examples in https://github.com/apache/camel-quarkus/tree/master/examples 
* Quarkus maven plugin and various IDE plugins

### PR DESCRIPTION
**Pull request by jamesnetherton**
Fixes #1824

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: docs/modules/ROOT/pages/user-guide/first-steps.adoc [hunk 1]
```diff
@@ -1,98 +1,111 @@
...
+* https://plugins.jetbrains.com/plugin/13234-quarkus-tools[IntelliJ plugin]
+* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[VSCode plugin]
+
+Check the plugin documentation to discover how to create projects for your preferred IDE.
+
+== Example projects
+
+Camel Quarkus provides a GitHub repository containing set of example projects.
+
+https://github.com/apache/camel-quarkus-examples
```
Comment (ppalaga) [inline docs/modules/ROOT/pages/user-guide/first-steps.adoc:59]: I wonder whether we should mention the "List of extensions" page in addition to https://github.com/apache/camel-quarkus-examples ? 

https://camel.apache.org/camel-quarkus/latest/user-guide/examples.html 

or `xref:user-guide/examples.adoc`

I think it gives a bit better overview of which examples are there compared to https://github.com/apache/camel-quarkus-examples

**Review by ppalaga — APPROVED** [review]
Great work, thanks @jamesnetherton !

A suggestion inline.

**Comment by jamesnetherton** [comment]
Latest commit adds an `xref` link.

**Comment by ppalaga** [comment]
Perfect, thanks!

**Pull request merged**